.github/workflows/depcheck.yml

```diff
@@ -0,0 +1,27 @@
+name: Depcheck
+
+on:
+  push:
+    branches:
+      - nextjs
+  pull_request:
+    branches:
+      - nextjs
+
+jobs:
+  check_dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
```

Comment:
sweep: we don't use node-version 14. Look in `.github/workflows/nodejs.yml`. Our react build check uses node version 18.